OxfordBodleian/Juel-Jensen/BDLaethe32.xml

```diff
@@ -822,55 +829,75 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                 </item>
-                                
+
                                 <item xml:id="e3">
-                                    <desc>A white sticker is glued to the inner side of the back board. 
-                                      A short description of the manuscript's content and main physical features is printed on it. Written 
+                                    <desc>A white sticker is glued to the inner side of the back board.
+                                      A short description of the manuscript's content and main physical features is printed on it. Written
                                       in pencil on the sticker are ‘Ge/x/-’, ‘PB 371’, ‘₤ 40’, and ‘1643’.</desc>
                                 </item>
                             </list>
                         </additions>
-                        
-                     <!--   <bindingDesc>
-                            <binding xml:id="binding" contemporary="false">
-                                <decoNote xml:id="b1">Modern binding</decoNote>
-                                <decoNote xml:id="b2" type="bindingMaterial">
-                           <material key="wood"/>
-                        </decoNote>
-                            </binding>
-                        </bindingDesc>-->
-                        
+
+                        <bindingDesc>
+                            <binding contemporary="Ethiopian">
+                            <decoNote xml:id="b1" type="bindingMaterial">
+                              The book is bound in <material key="wood">wooden</material> boards.
+                            </decoNote>
+                            <decoNote xml:id="b2" type="Boards">
+                              The boards are flush with the first and last quires but are slightly smaller than the central ones.
+                              The boards have external bevels and the grain runs vertically. Both outer and inner surfaces of the boards exhibit signs of the working tool.
+                              However, the outer surface and the edges are polished and a layer of dark-coloured finish has been added.
+                              It is possible that the lower board served as a support for other operations since on the inner side, small punched holes are visible.
+                            </decoNote>
+                            <decoNote xml:id="b4" type="SewingStations">4</decoNote>
+                            <decoNote xml:id="b5">
+                              Chainstitch sewing on four sewing stations (two pairs) executed with Z-plied threads; the sewing presents a periodic fold pattern except in the first quire
+                              where it is continuous and the the thread lenghts are double between the second and the third sewing station.
+                              The ends of the threads are knotted outside the last quire to the chains fomed by the sewing on the spine.
```

Comment:
"fomed" sounds weird to me. Is it correct?

Reply:
Indeed, it's formed!